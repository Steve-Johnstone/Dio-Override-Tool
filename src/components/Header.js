import React from 'react';
import 'bulma/css/bulma.min.css';

const Header = () => {
	return (
		<div className='is-flex is-justify-content-space-between'>
			<img
				className='logo'
				src='image/hotelscom-logo.png'
				alt='hcom logo'
				data-testid='logo'
			></img>
			<p>Dio Override Tool</p>
		</div>
	);
};

export default Header;
