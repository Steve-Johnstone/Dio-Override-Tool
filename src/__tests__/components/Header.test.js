import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Header from '../../components/Header';

describe('Header component', () => {
	it('should display the DOM correctly', () => {
		const tree = renderer.create(<Header />).toJSON();

		expect(tree).toMatchSnapshot();
	});
});
