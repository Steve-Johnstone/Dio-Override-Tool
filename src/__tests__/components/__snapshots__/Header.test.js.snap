// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should display the DOM correctly 1`] = `
<div>
  <p
    className="ml-6"
  >
    Dio Override Tool
  </p>
</div>
`;
